packages/protocol/migrations/22_governance.ts

```diff
@@ -62,7 +46,9 @@ module.exports = deploymentForCoreContract<GovernanceInstance>(
       'Escrow',
       'Exchange',
       'FeeCurrencyWhitelist',
+      'Freezer',
       'GasPriceMinimum',
+      'GoldToken',
```

Comment:
Should we also remove the elect validators migration?

Reply:
It's useful for testing purposes, and can be easily skipped as it's the very last migration. I think it's fine for now